app/src/main/java/com/courseproject/tindar/entities/Account.java

```diff
@@ -0,0 +1,311 @@
+package com.courseproject.tindar.entities;
+
+import android.location.Location;
+import java.util.ArrayList;
+import java.util.Date;
+
+@SuppressWarnings("FieldMayBeFinal")
+
+public class Account {
+    private boolean isActiveStatus;
+    private final String id;
+    private String email;
+    private String password;
+    private String displayName;
+    private String firstName;
+    private String lastName;
+    private String aboutMe;
+    private String profilePictureLink;
+    private ArrayList<String> matchList;
+    private ArrayList<String> likeList;
+    private ArrayList<String> blockList;
+    private ArrayList<String> recentlyViewed;
+    private Date birthdate;
+    private Location location;
+    private Gender gender;
```

Comment:
Can we have Location and Gender as String to make it simpler? The frontend will only give a few options to select from so frontend will control what values a user can enter for those two attributes.